server/api/src/db/requests/views.py

```diff
@@ -61,6 +63,9 @@ def create():
         ) WITH DATA;
 
         CREATE UNIQUE INDEX ON open_requests(srnumber);
+        CREATE INDEX ON open_requests(nc);
+        CREATE INDEX ON open_requests(requesttype);
+
     """)
```

Comment:
Right now we don't do any filtering on the `/open_requests` endpoint -- the "new" frontend wants all of them. So I'm not sure these new indexes are necessary, but also they probably don't hurt. We'd have to update `dev` and `prod` servers manually to add these. Let's discuss.

Reply:
same here. was trying some tuning. i'll back this out too.

Reply:
ok